.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          OPENAI_API_MODEL: "gpt-4o-mini"
```

Comment:
Switching the model from 'gpt-4-1106-preview' to 'gpt-4o-mini' may impact the quality and depth of the code review. Verify that 'gpt-4o-mini' meets the requirements for your code review process.